oneview_redfish_toolkit/api/enclosure_manager.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, enclosure, oneview_version):
 
         self.redfish["ManagerType"] = "EnclosureManager"
         self.redfish["Name"] = "Enclosure Manager"
+        self.redfish["Description"] = "HPE Onboard Administrator"
```

Comment:
This isn't really the OA that we are exposing in this service. Shouldn't the description be something more like 'Synergy Manager' or 'HPE OneView Manager'?

Reply:
same comment as above